<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>742</width>
    <height>481</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(30, 30, 30);
selection-background-color: rgb(48,140,198);
font: 11pt &quot;courier new&quot;;
color: white;
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTextEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>241</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(221, 221, 221);
selection-background-color
font: 13pt;
color: black;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>140</y>
      <width>401</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(147, 150, 157);
selection-background-color: rgb(30, 30, 30);
font: 75 12pt;
color: black;</string>
    </property>
    <property name="text">
     <string>Print</string>
    </property>
   </widget>
   <widget class="QComboBox" name="KeypresscomboBox">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>10</y>
      <width>76</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(221, 221, 221);
selection-background-color: rgb(30, 30, 30);
font: 12pt;
color: black;</string>
    </property>
    <item>
     <property name="text">
      <string>back</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>home</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="genButton4">
    <property name="geometry">
     <rect>
      <x>554</x>
      <y>10</y>
      <width>91</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(221, 221, 221);
selection-background-color: rgb(30, 30, 30);
font: 12pt;
color: black;</string>
    </property>
    <property name="text">
     <string>KeyPress</string>
    </property>
   </widget>
   <widget class="QPushButton" name="genButton5">
    <property name="geometry">
     <rect>
      <x>554</x>
      <y>40</y>
      <width>91</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(221, 221, 221);
selection-background-color: rgb(30, 30, 30);
font: 12pt;
color: black;</string>
    </property>
    <property name="text">
     <string>Scroll</string>
    </property>
   </widget>
   <widget class="QComboBox" name="ScrollComboBox">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>40</y>
      <width>76</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(221, 221, 221);
selection-background-color: rgb(30, 30, 30);
font: 12pt;
color: black;</string>
    </property>
    <item>
     <property name="text">
      <string>Up</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Down</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Right</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Left</string>
     </property>
    </item>
   </widget>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>721</width>
      <height>441</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(147, 150, 157);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPlainTextEdit" name="plainTextEdit">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>160</y>
       <width>701</width>
       <height>261</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>courier new</family>
       <pointsize>11</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(221, 221, 221);
selection-background-color
font: 10.5pt;
color: black;</string>
     </property>
    </widget>
    <widget class="QPushButton" name="genButton3">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>100</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(221, 221, 221);
selection-background-color: rgb(30, 30, 30);
font: 12pt;
color: black;</string>
     </property>
     <property name="text">
      <string>Wit+Clk</string>
     </property>
    </widget>
    <widget class="QPushButton" name="genButton2">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>20</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>courier new</family>
       <pointsize>12</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(221, 221, 221);
selection-background-color: rgb(30, 30, 30);
font: 12pt;
color: black;</string>
     </property>
     <property name="text">
      <string>UIWait</string>
     </property>
    </widget>
    <widget class="QPushButton" name="genButton">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>60</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(221, 221, 221);
selection-background-color: rgb(30, 30, 30);
font: 12pt;
color: black;</string>
     </property>
     <property name="text">
      <string>UIClick</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox">
     <property name="geometry">
      <rect>
       <x>590</x>
       <y>130</y>
       <width>121</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(181, 182, 183);
selection-background-color: rgb(30, 30, 30);
font: 12pt;
color: black;
</string>
     </property>
     <property name="text">
      <string>Auto Copy</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>121</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(147, 150, 157);
font: 75 12pt &quot;Courier New&quot;;
selection-background-color: rgb(30, 30, 30);
color: black;</string>
     </property>
     <property name="text">
      <string>Object Name</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="ui_text">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>50</y>
       <width>90</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(147, 150, 157);
font: 13pt &quot;courier new&quot;;
color: black;</string>
     </property>
     <property name="text">
      <string>Text</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="ui_name">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>80</y>
       <width>90</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(147, 150, 157);
font: 13pt &quot;courier new&quot;;
color: black;</string>
     </property>
     <property name="text">
      <string>Name</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="timeEdit">
     <property name="geometry">
      <rect>
       <x>640</x>
       <y>70</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="focusPolicy">
      <enum>Qt::StrongFocus</enum>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(221, 221, 221);
selection-background-color
font: 10pt;
color: black;</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <property name="verticalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarAsNeeded</enum>
     </property>
     <property name="html">
      <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'courier new'; font-size:11pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;7&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>550</x>
       <y>70</y>
       <width>91</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(147, 150, 157);
font: 75 12pt &quot;Courier New&quot;;
selection-background-color: rgb(30, 30, 30);
color: black;</string>
     </property>
     <property name="text">
      <string>Time(Sec)</string>
     </property>
    </widget>
    <widget class="QPushButton" name="genButton6">
     <property name="geometry">
      <rect>
       <x>450</x>
       <y>10</y>
       <width>91</width>
       <height>23</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(221, 221, 221);
selection-background-color: rgb(30, 30, 30);
font: 12pt;
color: black;</string>
     </property>
     <property name="text">
      <string>Sleep</string>
     </property>
    </widget>
    <zorder>label_3</zorder>
    <zorder>plainTextEdit</zorder>
    <zorder>genButton3</zorder>
    <zorder>genButton2</zorder>
    <zorder>genButton</zorder>
    <zorder>checkBox</zorder>
    <zorder>label</zorder>
    <zorder>ui_text</zorder>
    <zorder>ui_name</zorder>
    <zorder>timeEdit</zorder>
    <zorder>genButton6</zorder>
   </widget>
   <zorder>frame</zorder>
   <zorder>lineEdit</zorder>
   <zorder>label_2</zorder>
   <zorder>KeypresscomboBox</zorder>
   <zorder>genButton4</zorder>
   <zorder>genButton5</zorder>
   <zorder>ScrollComboBox</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>742</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>courier new</family>
     <pointsize>11</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <widget class="QMenu" name="menuTestPresso_API">
    <property name="font">
     <font>
      <family>courier new</family>
      <pointsize>11</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="title">
     <string>TestPresso API</string>
    </property>
   </widget>
   <addaction name="menuTestPresso_API"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
